### DISCUSSION TITLE
Bug when applying for github education benifits.

### DISCUSSION THREAD

**Discussion by Vishalthakur9645** [community/community#129967] [GitHub Education]
### Hi everyone,

Im Vishal a 19 year old student i just want to say that when i go to show my student id there is only one option of camera and when i click on it it only uses ny front camera cant use my back camera from front camera it is near to impossible to show id.
Here im aslo attaching a screenshot as you can see i have selected back camera but still it is using front camera.
Thank you please reply as soon as possible.
![Screenshot_20240627-161337_Edge.jpg](https://github.zendesk.com/attachments/token/4L0vc8Z5WeC11Gu9O3mg5FptK/?name=Screenshot_20240627-161337_Edge.jpg)

**Comment by maxime-bodifee** [comment]
Try the following steps, this information is from the [FAQ](https://github.com/orgs/community/discussions/111352#use-front-camera):

### I can only use my Front Camera to take a photo
We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.
